src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  $cell-side: 100px;
+  $borders: 1px;
+  max-width: 8 * $cell-side;
+  padding: $cell-side;
+  display: flex;
+  gap: $borders;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    height: $cell-side;
+    width: $cell-side;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: $borders solid black;
+    transition: transform 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__#{$i}::before {
+      content: "#{$i}";
+      font-size: 30px;
+      font-family: Arial, Helvetica, sans-serif;
+    }
```

Comment:
```suggestion
    &__#{$i}::before {
      content: "#{$i}";
    }
```
You can just use calendar__day selector for other properties
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/09e5be28-9a4e-4f61-80a2-bd28eaabb306)